openedx/features/enterprise_support/signals.py

```diff
@@ -99,7 +102,7 @@ def handle_enterprise_learner_subsection(sender, user, course_id, subsection_id,
             'grade': str(subsection_grade),
         }
 
-        transmit_subsection_learner_data.apply_async(kwargs=kwargs)
```

Comment:
Looking into the code for these two tasks `transmit_single_subsection_learner_data` and `transmit_subsection_learner_data` I am a bit confused by the comments. both say send data `to each linked integrated channel.` but looks like only the former does that, the second one only sends to one channel. maybe it's just a comment thing? and why the difference, one sends to all channels, and another sends to a single channel?

Reply:
huh... these are good points. I think `transmit_subsection_learner_data` should be iterating over the channels. I can add a PR for that.

Reply:
Hmm, actually looking at the existing `transmit_learner_data` task, it too says `"to each linked integrated channel"` but only grabs the one 🤔

Reply:
Oh I see, the command manages the channels for the bulk transmission tasks. Okay, I can ensure we're iterating in the bulk assessment level command.

Reply:
already done 😄 https://github.com/edx/edx-enterprise/blob/master/integrated_channels/integrated_channel/management/commands/transmit_subsection_learner_data.py#L56-L61

Reply:
yeah the comment maybe what is confusing us? also the difference in where the looping lives is what confused me, why we chose to design one with a channel id input, another without? but not for this PR of course

Reply:
I would maybe at least update the comment in the method `transmit_subsection_learner_data` https://github.com/edx/edx-enterprise/blob/31d5bc03a4aac954727f6328e242fa57e15e056a/integrated_channels/integrated_channel/tasks.py#L192